samples/grids/data-grid/column-types/src/DataGridColumnTypes.tsx

```diff
@@ -74,9 +74,9 @@ export default class DataGridColumnTypes extends React.Component<any, any> {
                     paddingLeft="10" paddingRight="0"
                         cellUpdating={this.onSalesCellUpdating} width="*>180"/>
 
-                    <IgrTemplateColumn field="Productivity" headerText="Productivity"
+                    {/* <IgrTemplateColumn field="Productivity" headerText="Productivity"
```

Comment:
please add comment: "TODO un-comment this column when editing templated columns is corrected"